server/viewModels/device.ts

```diff
@@ -4,7 +4,14 @@ import Location from '../data_models/location'
 
 type DeviceDetailViewModel =
   | BaseErrorModel
-  | (BaseErrorModel & { device: Device; backLink: string })
-  | (BaseSuccessModel & { device: Device; locations: Array<Location>; backLink: string })
+  | (BaseErrorModel & { device: Device; backLink: string; searchUrl: string })
```

Comment:
Have you confirmed the error model loads correctly given it doesn't have the date parameters passed at all?

Reply:
Cypress tests!!!